### ISSUE TITLE
[Lighthouse a11y] Inaccurate Error Suggestion for "Year" Edit Field.

### ISSUE THREAD

**Issue by shreyachatterjee00** [civiform/civiform#12889]
Describe the bug
Issue: Inaccurate Error Suggestion for "Year" Edit Field. JAWS announces "Please lengthen this text to 4 characters or more" and VO announces "Use at least 4 characters", but 4 is the max amount of characters you can input in the "Year" Edit Field.

Location: Comprehensive Sample Program Form Screen 1

WCAG Success Criterion
3.3.3 Error Suggestion

To Reproduce
1. Navigate to the "Year" Form Field.
 2. Input 1-3 characters.
 3. Select the "Continue" Button.

Expected behavior
Expectation: Error Suggestion announcement that is clear and logical.

Remediation:
Edit the Error Suggestion to something like "Enter a 4 digit year e.g. 1996."

**Comment by melanie-exygy** [comment]
The fix in PR #13636 is an improvement on the error message, but relying on the native browser error and `title` attribute isn't ideal because of the way some browsers handle it.  See this Slack thread:  https://civiform.slack.com/archives/C01R8T4EGGG/p1784574313164969.
